### PR TITLE
[FA] 12-1.md edit translation base on review

### PR DESCRIPTION
**Pull request by thisistayeb**

### CODE REVIEW

File: docs/fa/week12/12-1.md [hunk 1]
```diff
@@ -6,9 +6,7 @@ lecturer: Mike Lewis
 authors: Jiayu Qiu, Yuhong Zhu, Lyuang Fu, Ian Leefmans
 date: 20 Apr 2020
 translator: Tayeb Pourebrahim
-translation-date: 10 Oct 2020
```
Comment (Atcold) [inline docs/fa/week12/12-1.md:9]: Why removing this?

File: docs/_config.yml [hunk 2]
```diff
@@ -610,6 +610,7 @@ fa:
     - path: fa/week12/12.md
       sections:
         - path: fa/week12/12-1.md
+        - path: fa/week12/12-2.md
```
Comment (Atcold) [inline docs/_config.yml:613]: We already have this translation!

**Review by Atcold — CHANGES_REQUESTED** [review]
I'm not sure I understand what's going on.
`12-2.md` was already translated at #673.

File: docs/_config.yml (hunk 2 above)
Comment (thisistayeb) [reply docs/_config.yml:613]: I just edit some sentences.

**Comment by Atcold** [comment]
No idea what's going on here.

**Pull request closed without merging**